setup.cfg

```
@@ -35,3 +35,10 @@ test =
 
 [bdist_wheel]
 universal = 1
+
+[versioneer]
+VCS = git
+style = pep440-post
+versionfile_source = openfe/_version.py
+versionfile_build = openfe/_version.py
+tag_prefix = "" # see https://github.com/python-versioneer/python-versioneer/blob/master/INSTALL.md
```

This is causing issues
```suggestion
tag_prefix = '' # see https://github.com/python-versioneer/python-versioneer/blob/master/INSTALL.md
```

---

Actually I wonder if it's more the comment afterwards that is causing issues (given it's attempted to add quotes around it)

---

Sorry I 100% missed that this was probably intentional 🤦🏽‍♂️

---

Haha, I left that comment in there to remind myself when I made the PR to ask about it. I will fix it.